github/commitstatus.go

```diff
@@ -18,6 +18,7 @@ import (
 	"time"
 
 	"github.com/Pix4D/cogito/retry"
+	"github.com/bradleyfalzon/ghinstallation"
```

Comment:
seems small enough... but we need to check if its really crucial to bring in this dependency...